packages/forms/src/UIForm/fields/ListView/__snapshots__/ListView.component.test.js.snap

```diff
@@ -502,132 +499,101 @@ exports[`ListView field render should render no items message 1`] = `
       required={true}
       t={[Function]}
     >
-      <Header
-        headerDefault={
-          Array [
-            Object {
-              "icon": "talend-search",
-              "id": "my-list-view-search",
-              "label": "Search for specific values",
-              "onClick": [Function],
-            },
-          ]
-        }
-        headerLabel="Countries"
-        i18n={Object {}}
-        nbItems={0}
-        nbItemsSelected={0}
-        required={true}
-        t={[Function]}
-        tReady={true}
+      <header
```

Comment:
This complicated diff is in fact simple, just displayed in a non simple way.

Before
```
<withI18nextTranslation(Header)
   ... // bunch of props + withTranslation HOC injected props (i18n, tReady, t)
>
    <Header
        ... // same props as the HOC above
    >
        {content}
    </Header>
</withI18nextTranslation(Header)>
```

Now with the talend/scripts HOC mock we don't have this double level anymore. The HOC returns the component directly, with the 't()` and a changed displayName

After
```
<withI18nextTranslation(Header) // this is the Header with t() and a changed display name
>
    {content}
</withI18nextTranslation(Header)>
```

This pattern applies on every snapshot with a withTranslation HOC